### PR TITLE
[REC-2024] Initial Recife 2024 event

### PR DESCRIPTION
**Pull request by fabioqcorreia**
Including 2024 Recife event.

### CODE REVIEW

**Review by glasnt — APPROVED** [review]
Pending a second review from core for minor question

File: data/events/2024-recife.yml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+enddate: 2024-12-14T18:00:00-03:00
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: "2023-12-22T00:00:00-03:00" #2019-09-17T00:09:00-03:00 # start accepting talk proposals.
+cfp_date_end:  "2024-04-30T23:59:59-03:00" #2019-10-25T00:09:00-03:00 # close your call for proposals.
+cfp_date_announce: "2023-12-22T00:00:00-03:00" #2019-10-20T00:09:00-03:00 # inform proposers of status
+
+
+cfp_open: "true"
+cfp_link: "https://www.papercall.io/dodrec24" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
```
Comment (adrianmoisey) [inline data/events/2024-recife.yml:25]: This link doesn't seem to work: 
image

**Comment by adrianmoisey** [comment]
I'm happy to merge, but I don't want to merge without confirmation that the CFP URL is fine.

**Comment by fabioqcorreia** [comment]
Hi! The link is fine, we just didn't go live yet.
Sending a screenshot so you can be sure that everything works fine.
image

**Pull request merged**